components/channel_layout/channel_controller.tsx

```diff
@@ -18,16 +24,23 @@ import ProductNoticesModal from 'components/product_notices_modal';
 
 import Pluggable from 'plugins/pluggable';
 
-import {isInternetExplorer, isEdge} from 'utils/user_agent';
+import {shouldShowAppBar} from 'selectors/plugins';
 
-interface Props {
-    shouldShowAppBar: boolean;
-    fetchingChannels: boolean;
-}
+import {GlobalState} from 'types/store';
+
+import {Constants} from 'utils/constants';
+import {isInternetExplorer, isEdge} from 'utils/user_agent';
 
 const BODY_CLASS_FOR_CHANNEL = ['app__body', 'channel-view'];
 
-export default function ChannelController({shouldShowAppBar, fetchingChannels}: Props) {
+export default function ChannelController() {
+    const dispatch = useDispatch<DispatchFunc>();
+
+    const shouldRenderCenterChannel = useSelector((state: GlobalState) =>
+        state.requests.channels.getChannelsMembersCategories.status === RequestStatus.SUCCESS);
```

Comment:
isFetching prop which was there previously is now converted to its own selector and now being used here